dot/rpc/subscription/websocket.go

```diff
@@ -212,10 +213,32 @@ func (c *WSConn) initStorageChangeListener(reqID float64, params interface{}) (u
 	return myObs.id, nil
 }
 
-func (c *WSConn) unsubscribeStorageListener(params interface{}) {
-	id := params.([]interface{})[0].(float64)
-	observer := c.Subscriptions[uint(id)].(state.Observer)
+func (c *WSConn) unsubscribeStorageListener(reqID float64, params interface{}) {
+	switch v := params.(type) {
+	case []interface{}:
+		if l := len(v); l < 1 {
+			c.safeSendError(reqID, big.NewInt(-32600), "Invalid request")
+			return
+		}
+	default:
+		c.safeSendError(reqID, big.NewInt(-32600), "Invalid request")
```

Comment:
Declare this as constant
```
big.NewInt(-32600)
```

Reply:
I can't make big.NewInt(-32600) a const, so I've made -32600 a const and I'm using that.